packages/lodestar/src/chain/blocks/pool.ts

```diff
@@ -27,25 +28,42 @@ export class BlockPool {
     this.forkChoice = forkChoice;
   }
 
+  public async start(): Promise<void> {
+    this.eventBus.on("clock:slot", this.onNewSlot);
```

Comment:
Can you add this handler in src/chain/chain.ts instead? This will require the block pool to be exposed to the chain.

I'd like to capture all event subscriptions there and prevent these submodules from growing start/stop methods. In a future PR, I'd even like to remove this `eventBus` and `forkChoice` dependencies, as the "unknown block root" scenario can/will be handled by a certain `BlockError` outcome.